README.md

```diff
@@ -1,3 +1,68 @@
 [![Build Status](https://travis-ci.org/GoogleCloudPlatform/distroless.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/distroless)
 
-# This repository contains source code used to build "distroless" docker images.
+# "Distroless" Docker Images
+
+"Distroless" images contain only your application and it's runtime dependencies.
+They do not contain package managers, shells any other programs you would expect to find in a standard Linux distribution.
+
+For more information, see this talk.
```

Comment:
You can link to this for now, once the video is posted we can link to that:
https://swampup2017.sched.com/event/A6CW/distroless-docker-containerizing-apps-not-vms?iframe=no&w=100%&sidebar=yes&bg=no